topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio/tutorial.md

```diff
@@ -366,11 +365,11 @@ filtered_srt<- SCTransform(filtered_srt, vars.to.regress = c("perc.mt", "nFeatur
 {: .comment}
 
 
-We also have loads of genes, but not all of them vary in expression from cell to cell. For instance, housekeeping genes are defined as not changing much from cell to cell, so we could remove these from our data to simplify our analyses.
+We also have loads of genes, but not all of them vary in expression from cell to cell. For instance, housekeeping genes are defined as not changing much from cell to cell, so we could disclude these from our data to simplify our analyses.
```

Comment:
> disclude

genuinely had no idea this was a real word!

Reply:
exclude might be more familiar to an average reader? but likewise, til!

Reply:
I also have no idea that this word exists. Did we agree on keeping it? :smile:  otherwise, ready to merge